app/controllers/home_controller.rb

```diff
@@ -157,4 +159,9 @@ def activity
     ]
     response
   end
+
+  def trending
+    exclude_tids = @tag_subscriptions.pluck(:tid)
+    Tag.trending.where.not(tid: exclude_tids)
```

Comment:
very nice!!! so `@tag_subscriptions` is assumed, rather than using a parameter? It might be good to be explicit since this would fail without an existing value for `@tag_subscriptions`, right? But it's fine with me for now; maybe this could be filed as a follow-up PR?

Reply:
Yes, that's a valid point...I'll create a follow up Issue/PR for this

Reply:
Hi @jywarren 👋🏾 

I tested this out locally and it seems that it does not break if the `@tag_subscriptions` is empty... it returns an empty array...
Tested below...
<img width="615" alt="Screenshot 2021-01-20 at 21 22 03" src="https://user-images.githubusercontent.com/7622875/105217942-8cfac000-5b65-11eb-89e6-c5e3f95c8228.png">

However, I am assuming that maybe it would be best to have a sort of 'fail safe' if `@tag_subscriptions` is empty...
I came up with this code...let me know if this makes sense... I am not sure if it is redundant...because we know what will happen if the `@tag_subscriptions` is empty

```
  def trending
    exclude_tids = @tag_subscriptions.present? ? @tag_subscriptions.pluck(:tid) : []
    Tag.trending.where.not(tid: exclude_tids)
  end
```